components/Badge/index.html

```diff
@@ -0,0 +1,100 @@
+            <a class='li-link'href='../../index.html'> <h3>Chikku<span style='color:rgb(10, 177, 10)'>Ui</span></h3> </a>  
+        </div>
+        <div class='list-parent-right' id='modal'>
+            <a  href="javascript:void(0);" class="li-icon" onclick="modelFunction()">
+                <i class="fa fa-bars"></i>
+              </a>
+              <a class='li-link' href='../../index.html'> Home</a>
+              <a class='li-link' href='components/alert/index.html'>Documentation</a>
+              <a class='li-link' target='_blank' href='https://github.com/vineeth-t/Main/tree/Devlopment'>Github</a>
+              <!-- <a class='li-link'href='#'>V0.1</a> -->
+        </div>
+        </nav>
+    <header class='header' >
+        <div class='menu-list' id='menu-items'>
+            <a class="li-link component" href='/components/alert/index.html'>Alert</a>
+            <a class="li-link" href='/components/Avatar/index.html'>Avatar</a>
+            <a class="li-link link-active" href='/components//Badge/index.html'>Badge</a>
+            <a class="li-link " href='/components/Buttons/index.html'>Button</a>
+            <a class="li-link" href='/components/Card/index.html'>Cards</a>
+            <a class="li-link" href='/components/List/index.html'>Lists</a>
+            <button class='Float-btn Floating-btn-x' onclick="floatingOff()">X</button>
+            
+        </div>
+        <div class='component-content'>
+            <button class='Float-btn Floating-btn' onclick="floatingAction()"><></button>
+            <h2>Badge</h2>
+            <p class='content-txt'>Css badges are easy to build and used to tell the current status of any specific user or tell no of items in a cart</p>
+            <hr><br/>
+            <h2>Badge on Avtar </h2>
+            <p class='content-txt'>Hover to see status</p>
+            <div class='bdg-av'>
+                <div class='badge-on-avtar'>
+                    <img class='avtar' src='https://pbs.twimg.com/profile_images/1348136483828350983/tKCPXjZD_400x400.jpg' alt='Profile Photo'/>
+                    <div class='badge online'></div>
+                    <div class='badgeStatus'>Online</div>
+                </div>
+                <div class='badge-on-avtar'>
+                    <img class='avtar' src='https://pbs.twimg.com/profile_images/1348136483828350983/tKCPXjZD_400x400.jpg' alt='Profile Photo'/>
+                    <div class='badge offline'></div>
+                    <div class='badgeStatus'>Offline</div>
+                </div>
+                <div class='badge-on-avtar'>
+                    <img class='avtar' src='https://pbs.twimg.com/profile_images/1348136483828350983/tKCPXjZD_400x400.jpg' alt='Profile Photo'/>
+                    <div class='badge onCall'></div>
+                    <div class='badgeStatus'>Attending a Call</div>
+                </div>
+
+            </div>
+                <div>
+                    <!-- <iframe
```

Comment:
can remove this , and do check if any more unused code is there in any file,  you can remove that

Reply:
Hey. I'm planning to add Iframe to my library in coming updates .The code will be used in next versions so kindly ignore this for now.Thanks